virtcontainers/kata_agent.go

```diff
@@ -46,6 +46,7 @@ var (
 	kataGuestSandboxDir   = "/run/kata-containers/sandbox/"
 	type9pFs              = "9p"
 	vsockSocketScheme     = "vsock"
+	vSockPort             = 1024
```

Comment:
OOI, why 1024? That's the first registered port number so I wonder if this should be 49152 (the first ephemeral port https://en.wikipedia.org/wiki/Ephemeral_port)?